src/main/web/templates/auth-needed.ftl

```diff
@@ -0,0 +1,3 @@
+<html>
```

Comment:
Since this is an example, ideally we'd use complete HTML (that will pass validation) if possible. Same would go for each example page. 

```
<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <title>Authentication Needed</title>
  </head>
  <body>
    You are authenticated!
  </body>
</html>
```